sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/HospitalizationTabPage.java

```diff
@@ -49,4 +49,10 @@
   public static final By BLUE_ERROR_EXCLAMATION_MARK =
       By.xpath("//span[@class='v-errorindicator v-errorindicator-info']");
   public static final By BLUE_ERROR_EXCLAMATION_MARK_TEXT = By.xpath("//div[@class='gwt-HTML']");
+  public static final By PLACE_OF_STAY_IN_HOSPITAL_POPUP =
+      By.xpath("//*[contains(text(),'Place of stay in hospital')]");
```

Comment:
cannot be taken by other identifier? to make it reusable also for DE if needed

Reply:
if needed we can make selector like PLACE_OF_STAY_IN_HOSPITAL_POPUP_DE, i didn't see any german test for that .. yet